mysql-test/suite/innodb/r/innodb-index-online.result

```diff
@@ -543,6 +542,31 @@ CHECK TABLE t1;
 Table	Op	Msg_type	Msg_text
 test.t1	check	status	OK
 DROP TABLE t1;
+#
+# MDEV-36281 DML aborts during online virtual index
+#
+CREATE TABLE t1(f1 INT NOT NULL PRIMARY KEY, f2 INT NOT NULL,
+f3 INT NOT NULL, f4 INT AS (f3) VIRTUAL,
+f5 INT AS (f1) VIRTUAL, INDEX(f4))ENGINE=InnoDB;
+INSERT INTO t1(f1, f2, f3) VALUES(1, 2, 3);
+SET DEBUG_SYNC = 'innodb_inplace_alter_table_enter SIGNAL dml_start WAIT_FOR dml_finish';
+ALTER TABLE t1 ADD INDEX v1(f5, f2, f4), ADD INDEX v2(f3, f5);
+connection con1;
+set DEBUG_SYNC="now WAIT_FOR dml_start";
+UPDATE t1 SET f3= f3 + 1;
+set DEBUG_SYNC="now SIGNAL dml_finish";
+disconnect con1;
+connection default;
+CHECK TABLE t1 EXTENDED;
+Table	Op	Msg_type	Msg_text
+test.t1	check	status	OK
+SELECT * FROM t1 USE INDEX(v1);
+f1	f2	f3	f4	f5
+1	2	4	4	1
+SELECT * FROM t1 USE INDEX(v2);
+f1	f2	f3	f4	f5
+1	2	4	4	1
```

Comment:
I think that we should try to use a covering index read for these: not requesting `*` but only those columns that are present in the secondary index. Can you check that `row_search_mvcc()` will not access the clustered index?